src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2654,7 +2654,10 @@ export interface IEditorStickyScrollOptions {
 	 * Maximum number of sticky lines to show
 	 */
 	maxLineCount?: number;
-
+	/**
+	 * Model to choose for sticky scroll by default
+	 */
+	defaultModel?: 'Outline Model' | 'Folding Provider Model' | 'Indentation Model';
```

Comment:
We typically use values without space. Lowercase and camelCase
E.g. foldingStrategy: `auto` | `indentation`
`files.autoSave`: `afterDelay`, `onFocusChange`